src/components/ActionNetworkWarning.jsx

```diff
@@ -1,30 +1,59 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import NetworkChangeGuide from './NetworkChangeGuide';
+import { Button } from 'antd';
+import Web3 from 'web3';
+import config from '../configuration';
 
 /**
  * It is used to show in modals when user wants to take an action while connected to wrong network.
  */
 
 function ActionNetworkWarning(props) {
-  const { incorrectNetwork, networkName } = props;
+  const { incorrectNetwork, networkName, switchNetwork, web3, isInline } = props;
```

Comment:
Why we are passing such amount of parameters to this component? I mean all the network parameters can be summarized in one required chain id, the the component itself obtains all others (web3, network name, etc.) itself.